apps/cli/src/logging/custom.ts

```diff
@@ -34,7 +34,12 @@ export function getNpmLogger(session: ISession): LoggerDE {
     error(data: string) {
       const line = data.trim();
       if (!line) return;
-      if (line.includes('deprecated') || line === 'npm' || line.includes('WARN')) {
+      if (
+        line.includes('deprecated') ||
+        line.includes('package is no longer supported') ||
```

Comment:
This silences the crypto deprecation warning

Reply:
It is baked into some Jupyter dependencies, so yes, but might take a while for updates to happen.